control/control_test.go

```diff
@@ -815,9 +815,6 @@ func TestMetricConfig(t *testing.T) {
 			errs := c.subscriptionGroups.validateMetric(m1)
 			So(errs, ShouldBeNil)
 		})
-		Convey("So mock should have name: bob config from defaults", func() {
-			So(c.Config.Plugins.pluginCache["0"+core.Separator+"mock"+core.Separator+"1"].Table()["name"], ShouldResemble, ctypes.ConfigValueStr{Value: "bob"})
-		})
```

Comment:
It wasn't valid here since the subscription group would need to be processed before the default would be applied to the plugin config cache.  I added the test back as part of the convey that runs `subscriptionGroups.ValidateMetric`.